verl/trainer/main_generation.py

```diff
@@ -45,7 +45,7 @@ def run_generation(config) -> None:
 
     if not ray.is_initialized():
         # this is for local ray cluster
-        ray.init(runtime_env={'env_vars': {'TOKENIZERS_PARALLELISM': 'true', 'NCCL_DEBUG': 'WARN'}})
+        ray.init(runtime_env={'env_vars': {'TOKENIZERS_PARALLELISM': 'true', 'NCCL_DEBUG': 'WARN'}}, num_cpus=1)
```

Comment:
every ray remote call is scheduled by raylet. can we make it an option that does not affect the default behavior. if slurm users come across this issue, we can recommend the config in the FAQ page. by default we just set it to null